core/domain/role_services.py

```diff
@@ -209,9 +208,10 @@ def get_role_graph_data():
     role_graph['links'] = []
     role_graph['nodes'] = {}
     for role in PARENT_ROLES:
-        role_graph['nodes'][role] = HUMAN_READABLE_ROLES[role]
-        for parent in PARENT_ROLES[role]:
-            role_graph['links'].append({'source': parent, 'target': role})
+        if role != feconf.ROLE_ID_SUPER_ADMIN:
```

Comment:
Shouldn't we just be removing ROLE_ID_SUPER_ADMIN altogether from the role system?